lib/cppcheck.cpp

```diff
@@ -1353,6 +1353,75 @@ void CppCheck::getErrorMessages()
     Preprocessor::getErrorMessages(this, &s);
 }
 
+void CppCheck::analyseClangTidy(const ImportProject::FileSettings &fileSettings )
+{
+    std::string allIncludes = "";
+    std::string allDefines = "-D"+fileSettings.defines;
+    for (const std::string &inc : fileSettings.includePaths) {
+        allIncludes = allIncludes + "-I""" + inc + """ ";
```

Comment:
The code `"-I"""` is the same as `"-I"` but I prefer the latter.

Reply:
And instead of `""" "` I prefer `" "`

Reply:
no the extra " are used for folders that contain spaces. Something like -I"/test folder"

Reply:
I don't know what nonportable tweaks your compiler has ... but the portable way to write that would be :

    "-I\"" + inc + "\" "